### ISSUE TITLE
Elaboration of Tripaille documentation?

### ISSUE THREAD

**Issue by RichardBruskiewich** [galaxy-genome-annotation/python-tripal#7]
Colleagues,

After following the link off the docker-tripal page in the README near _"..If you are using Tripaille to load data..."_, I landed inside this project.

Sorry for being a bit pedantic here - I know I can perhaps read between the lines - but perhaps the README of this project could be prefaced with a more complete definition about **exactly what is Tripaille** (I know it is a CLI program), what is its purpose and scope of action (something to do with data loading, but...)? The name 'tripaille' looks like some kind of acronym starting with 'Tripa" (from "Tripal") but perhaps, the suffix letters mean something descriptive of the scripts purpose in life... or, maybe it is just a fanciful name?

In other words, give the newby's a soft landing in this project so they quickly get their bearings...

thanks.
Richard

**Comment by abretaud** [comment]
I've added more description in the README, is it better like this?
